src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/model/ProjectCreationModel.kt

```diff
@@ -72,7 +72,12 @@ class ProjectCreationModel {
         if (collectionStore.size > 1) {
```

Comment:
Does IntelliJ suggest this should be converted to a `when`? If so, maybe make that change.
```
when {
   collectionStore.size > 1 -> {}
   /* etc */
}
```

Reply:
yes it does!

Reply:
usually I would see it recommend 
```
if(collectionStore.isNotEmpty()) {
...
}
```

Reply:
@sarabiaj I think that would be if `if (collectionStore.size > 0)`